test/README.md

```diff
@@ -303,6 +303,15 @@ Use the `-v` option for verbose output.
 
 #### Dependencies
 
+##### Shell
```

Comment:
You could changes this from Shell to Bash. This would help hint that a macOS user should switch from zhs to bash.

Reply:
done, thanks!